2023-11-28 Move compensation changes process steps to the BizOps page from the Communications page (#15316)
fleetdm/fleet: handbook/business-operations/README.md

| Accelerations (quarterly)    | "Off-cycle" payroll          | Head of Revenue Operations
| [US contractor payroll](https://fleetdm.com/handbook/business-operations#run-us-contractor-payroll) | "Off-cycle" payroll | Head of Business Operations

### Compensation changes
Fleet evaluates and (if necessary) updates compensation decisions yearly, shortly after the anniversary of a team member's start date (see [Workiversaries](https://fleetdm.com/handbook/company/communications#workiversaries). The process for that evaluation and update is:
- On the first Friday of the month, the Head of BizOps posts in the `#help-classified` channel with the list of teammates celebrating anniversaries over the next month.
- On the day of the fleetiversary, the Head of BizOps or executive team member will post in `#random` celebrating the tenure of the teammate.

> - The Head of BizOps confers with manager or head of department and prepares compensation benchmarking data and schedules time with the CEO and CTO over an existing 1:1 to discuss if an adjustment needs to be made to compensation.

Reviewer: ```suggestion
- Schedule 30m to meet with the team member's manager
- Prepare latest compensation benchmarking data in ["Compensation decisions"](TODO: link)
- Schedule 30m with the CEO and CTO over an existing 1:1 to discuss if an adjustment will be made to compensation.
```

- During the 1:1 call, founders review benchmarking for role and geography, and decide if there will be an adjustment.

> - Head of BizOps posts to slack in `#help-classified` with the decision on compensation changes and effective date, if any.

Reviewer: imperative mood here and in all the places avoid answering who is doing what-

i.e. 

> Post to Slack in `#help-classified` with…

> - If a change is to be made, the Head of BizOps communicates decision to the teammate's people manager, who then communicates to their teammate.

Reviewer: ```suggestion
- Ask the team member's manager to communicate the decision about compensation to their teammate.
```

> - Head of BizOps updates the respective payroll platform (Gusto or Plane) and [equity spreadsheet](https://docs.google.com/spreadsheets/d/1_GJlqnWWIQBiZFOoyl9YbTr72bg5qdSSp4O3kuKm1Jc/edit?usp=sharing) (internal doc).

Reviewer: todo: detailed instructions for exactly what to update in equity plan, and a warning about what _not_ to edit

> - If an additional equity grant is part of this compensation change, the previous equity and new situation is noted in detail in the "Notes" column of the equity plan, and the "Grant started?" column is set back to "todo" to add it to the queue for the next time grants are processed (quarterly).

Reviewer: ```suggestion
- If an additional equity grant is part of this compensation change, note the previous equity (number of shares) in the "Notes" column of the equity plan and set the "Grant started?" column back to "todo" to add it to the queue for the next time grants are processed.
```
